src/products/mozilla_org/manage_mozillaorg.md

```diff
@@ -0,0 +1,95 @@
+# Managing Mozilla.org
+
+Since the migration from .lang to Fluent, the L10n PM’s job is a lot simpler. The PM’s primary focus is on reviewing the string extractions provided by the bedrock team, provides feedback, manages the project through Pontoon and communicates important information to the communities.
+
+## Reviewing the PR
+
+When a PR is ready for localization review, through a script, the PR is pushed to the [www-l10n](https://github.com/mozilla-l10n/www-l10n/tree/master/) repository. The naming convention of the string ID is the file name followed by the first few words of the string, up to 7 words.
```

Comment:
It's a good thing that I have no clue how the process works, because that makes me a good target for the do. Sadly, the document really doesn't help me understand the process.

* What's "the PR"? Who creates them? What is "www-l10n"? 
* Why are you bringing up the naming convention here? This should be in its own section, or in a "general issues to look out for" section.